wp/lib/bap_wp/tests/unit/test_compare.ml

```diff
@@ -123,7 +123,7 @@ let test_sub_pair_1 (test_ctx : test_ctxt) : unit =
   let output_vars = Var.Set.singleton z in
   let compare_prop, _, _ = Comp.compare_subs_eq
       ~input:input_vars ~output:output_vars
-      ~original:(sub1,env1) ~modified:(sub2,env2) in
+      ~original:(sub1,env1) ~modified:(sub2,env2) ~smtlib_post:"" ~smtlib_pre:"" in
```

Comment:
Should `smtlib_post` and `smtlib_pre` be optional arguments defaulting to `""`? This one is really up to you.

Reply:
Yeah. i debated this one. I just hated putting them as the first arguments, which optional seems to require. But it might be the right call

Reply:
I vote for `true` and `false` as defaults, respectively.

Reply:
Shouldn't the defaults both be `true`?

Reply:
Yes, I think you're right, actually.